docs/docs/07-plugins.md

```diff
@@ -35,6 +35,11 @@ This is all you need to add Babel to your application build pipeline. If the plu
 }
 ```
 
+The **order** of plugins is important, as for example, if there are multiple plugins that load/build particular type of file, the first matching will take precedence.
+
+To make a plugin available, you have to put it in your project `devDependency` list which will install it locally and make it available to snowpack.
```

Comment:
I think adding this is helpful! We skip over install instructions. But what about instead a) moving this to the very top of the section, and b) outlining the `npm install` command more clearly (I’m worried that some more beginner readers may not know how to “put it in your project `devDependency` list”)? 

```bash
npm install --save-dev @snowpack/[plugin-name]
```

Reply:
Sure, was uncomfortable to expand too much :)